src/features/Dashboard/components/TransactionLabel.tsx

```diff
@@ -37,24 +27,20 @@ const txStatusConfig: ITxStatusConfig = {
   [ITxStatus.CONFIRMED]: { color: '#75b433' }
 };
 
-const SStage = styled(Typography)`
-  color: ${(p: { color: string }) => p.color || 'transparent'};
-`;
-
 export default function TransactionLabel({ image, label, stage, date }: Props) {
   const config = txStatusConfig[stage];
   return (
-    <div className="TransactionLabel">
+    <Box variant="verticalCenter">
       {image}
-      <div className="TransactionLabel-info">
-        <Typography className="TransactionLabel-info-label">{label}</Typography>
-        <SStage color={config.color} className="TransactionLabel-info-stage">
-          {capitalize(stage)}{' '}
-        </SStage>
-        <Typography className="TransactionLabel-info-Date">{`${
-          date ? ` - ${formatDateTime(date)}` : ''
-        }`}</Typography>
+      <div>
+        <Body as="span">{label}</Body>
+        <div>
+          <Body as="span" color={config.color || 'transparent'} capitalize={true}>
```

Comment:
Why `transparent`?

Reply:
It was the default on the styled-component that was removed, so including it here as well.